### PR TITLE
Makes corrections suggested by lgtm.

### PR DESCRIPTION
**Pull request by obrien951**
## Description
Fixed a few lgtm errors

## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

File: psi4/driver/procrouting/dft/superfunctionals.py [hunk 1]
```diff
@@ -221,7 +221,7 @@ def test_ccl_functional(functional, ccl_functional):
         gamma_bb[index] = point['gamma_bb']
         index = index + 1
 
-    super = build_superfunctional(functional, N, 1)
+    super = build_superfunctional(functional, True, N, 1)
```
Comment (dgasmith) [inline psi4/driver/procrouting/dft/superfunctionals.py]: Can you explicitly pass kwargs here? `(functional, True, npoints=N, deriv=1)`

Comment (loriab) [reply psi4/driver/procrouting/dft/superfunctionals.py]: @obrien951, please edit when you get a chance.

File: psi4/driver/p4util/procutil.py [hunk 2]
```diff
@@ -280,19 +280,14 @@ def extract_sowreap_from_output(sowout, quantity, sownum, linkage, allvital=Fals
...
             if not line:
                 if E == 0.0:
-                    if allvital:
-                        raise ValidationError(
-                            'Aborting upon output file \'%s.out\' has no %s RESULT line.\n' % (sowout, quantity))
-                    else:
-                        ValidationError(
-                            'Aborting upon output file \'%s.out\' has no %s RESULT line.\n' % (sowout, quantity))
+                    raise ValidationError(
+                        'Aborting upon output file \'%s.out\' has no %s RESULT line.\n' % (sowout, quantity))
```
Comment (andysim) [inline psi4/driver/p4util/procutil.py]: Not really a problem with what you've done in this PR, but I think we should try to settle on a standard for `"xxx"%(yyy)` vs `"xxx".format(yyy)`.  I think it's fair to say that the latter is becoming more canonical, and I'd prefer that be used for the two instances in this PR where the `%` version is used.  The `.format` is present up above, I noticed.

Comment (andysim) [reply psi4/driver/p4util/procutil.py]: I welcome @loriab 's thoughts on that matter, but I suspect I already know that answer

Comment (dgasmith) [reply psi4/driver/p4util/procutil.py]: I might have been the last Python `%` holdout and have switched to format for the QCArchive ecosystem.

Comment (loriab) [reply psi4/driver/p4util/procutil.py]: I'm definitely in favor of f-string > `.format` > `%s` in general. Been gleefully moving to f-strings whenever it'll shorten the code. But these fns in particular are dead code for sow/reap that last worked in py2, so I was only willing to edit enough to silence lgtm.

Comment (andysim) [reply psi4/driver/p4util/procutil.py]: If this is dead code, there's no need to change anything then. Just wanted to bring it up

**Pull request merged**